forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,111 @@
+if (typeof CKEDITOR != 'undefined') {
+    CKEDITOR.on( "instanceReady", function() {
+        const codeBlock = document.getElementById( "cke_43" );
+        if ( location.href.includes( "/ask" ) ) {
+            codeBlock.addEventListener( "click", function() {
+                const category_1 = $('#category_1 option:selected');
+                if(category_1.text() === 'Programowanie') {
+                    const category_2 = $('#category_2 option:selected');			
+                    switch(category_2.text()){
+                        case 'HTML i CSS':

```

Comment:
Jeśli nie już wspomnianą tablicą, to można zrobić mapę (obiekt), gdzie kluczem będzie warunek przy CASE, a wartoscią string, który umieszczasz w propertisie **syntaxhighlight_lang**, wtedy jedną linijką załatwiasz sprawę, a mapa może być uniwersalna dla wszystkich switchy (które by były już zbędne).